ui/raidboss/data/04-sb/alliance/orbonne_monastery.js

```diff
@@ -707,7 +707,6 @@
       'replaceSync': {
         'Agrias': 'Agrias',
         'Aspersory': 'Aspersorium',
-        'Auracite Shard': 'Auracitenscherbe',
```

Comment:
Sorry, this is hard for me to review deletions when `missingTranslations` is true.  It is not clear to me without a lot more work whether you are removing translations that are required or translations that are unused.

I am sorry to request something tedious, but can you not remove translations from any section that is still missing translations? If it would be less work for you, I would be happy to file an issue and add some script code that would give warnings or errors on unused translations in these cases so you could find them automatically rather than re-doing this patch by hand.

If it would be helpful to prioritize translation issues that are most useful to cactbot, I would suggest looking at it with this view:
* highest priority: anything missing from `find_missing_translations.py` (other than a7s)
* medium priority: removing the `missingTranslations: true` from any section
* lowest priority: removing unused translations
* please don't do at all: changing capitalization of syncs

Reply:
All of the removals are elements that are neither part of a regex or are not in the txt version of the fight.

The reason behind it, is to avoid translating to much unaccassary things as some of them (not all) have a lot of incorrect stuff (e.g. elements from other fights)

Reply:
I don't disagree with removing.

However, my primary worry is that I don't want to remove an element that is currently in use, because that would make the state of translations worse.  When you are removing a bunch of lines from a timeline that has `missingTranslations: true` that it is impossible for me to know without verifying each of them individually whether or not that element is truly in use or not.  Even if you are doing it with a script, how do I know your script is correct? For sections where they are `missingTranslations: false`, then it is very easy to verify because the test would complain if it was in use.

I think it would be strictly better (for me, reviewing the pull request) to remove `missingTranslations: true` first, and then after that come back and remove unnecessary lines.

Sorry to be so stubborn on pull requests lately.  I just only have so much energy and time for reviewing, and so trying to figure out how to make things be as easy to read and verify as possible is important.

Reply:
Ok than i can't support other languages anymore since i can't fix the missing translations for other languages (if they are not from the gamefiles directly). 

If it might help you (thats what i did to somehow veryfi the changes in the end) i just removed all `'missingTranslations': true,` from all files in 04-sb (easy to do with sublime search replace just in a specified folder) and compare the reported results with the removals.

Reply:
Hmm.  That approach seems plausible but it will still take me a while to go through them all to verify.  I can still do it, it's just a lot.

(The problem here is that your script is taking all the translation lines that exist and translating them, and so if there's unnecessary ones then it can't do it? Could it just skip things it can't find from the data files? I guess it would help me to understand what the issue with leaving these lines in is.)

Reply:
Sorry also for all the push back on pull requests lately.  I hope you understand how much your contributions are appreciated.  This is another case where I mostly don't have enough information about how your process and script works and so I'm trying to find some middle ground where you can automatically translate things and it is not hard for me to review those translations.

Reply:
the script i am using is a modification of your `check_translation.py`.

It is not 100% optimal because in very, very rare cases it has false positives in it (i think there was a case with infernal fetters and fetters, which i had to look up in more detail)